src/plugins/share/public/components/url_panel_content.tsx

```diff
@@ -60,11 +60,20 @@ enum ExportUrlAsType {
 interface State {
   exportUrlAs: ExportUrlAsType;
   useShortUrl: boolean;
+  showTopNavMenu: boolean;
```

Comment:
Since this component is to be reused across Kibana, I'm not a big fan of bloating it with app-specific props. Can we make a HOC component around `UrlPanelContent`?